src/stores/utils/web3.js

```diff
@@ -76,6 +77,42 @@ export const estimateGas = async (web3, to, gasPrice, from, value, data) =>{
   return Web3Utils.toHex(gas.toString())
 }
 
+export const fromDecimals = (number, decimals) => {
```

Comment:
@branciard Do you think it would be good to move all the logic related to `fromDecimals` and `toDecimals` to another file? Something like `src/stores/utils/decimals.js`?

Reply:
I separate in decimal.js file in 6669067. Thank you for the PR feedback.